### PR TITLE
Feature/bottom navigation dropdown menu

### PR DESCRIPTION
**Pull request by JohnTomasTaylor**
- Added a "More" button on the bottom bar. This button opens up a drop down menu with more navigation options such as "Profile", "My Tups", etc... For now the "My Tups", "My Recipe" and "My events" options navigate to the recipefeed as their screens are being implemented.

- The PR also fixes a small bug in the login sceen where a failed login would make the "Sign In" button load indefinitely.
- Deleted overview screen so some features should be added to the main navigation promptly

### CODE REVIEW

**Comment by Tachi-67** [comment]
we already fixed the login screen bug in the chat pr ;)

**Review by dayan9265 — APPROVED** [review]
Nice job, this is a good addition to our app to make it feel more refined :). Apart from small details about the tests, the code looks good 👍

File: app/src/androidTest/java/ch/epfl/sdp/cook4me/ui/navigation/NavigationBarTest.kt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+class NavigationBarTest {
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<ComponentActivity>()
+
+    private val navigateTo = { expectedRoute: String ->
+        { actualRoute: String ->
+            assertThat(actualRoute, `is`(expectedRoute))
+        }
+    }
```
[the remark this answers is not in the document]
Comment (JohnTomasTaylor) [reply app/src/androidTest/java/ch/epfl/sdp/cook4me/ui/navigation/NavigationBarTest.kt:26]: Yes nicely spotted

File: app/src/androidTest/java/ch/epfl/sdp/cook4me/ui/navigation/NavigationBarTest.kt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    fun clickingOnMoreButtonShouldDisplayDropDownMenu() {
+        composeTestRule.setContent {
+            BottomNavigationBar(
+                navigateTo = navigateTo(""),
+                currentRoute = "",
+                onClickSignOut = {},
+            )
+        }
+        composeTestRule.onNodeWithStringId(R.string.bottom_bar_more_button_text).performClick()
+        composeTestRule.waitUntilExists(hasText(BottomNavScreen.Profile.title))
```
[the remark this answers is not in the document]
Comment (JohnTomasTaylor) [reply app/src/androidTest/java/ch/epfl/sdp/cook4me/ui/navigation/NavigationBarTest.kt:80]: Yes you are absolutely right

**Pull request merged**